src/components/[guild]/JoinModal/hooks/useJoin.tsx

```diff
@@ -80,9 +80,11 @@ const useJoin = (onSuccess?: () => void) => {
 
       addDatadogAction(`Successfully joined a guild`)
 
-      mutate(`/user/membership/${account}`)
-      // show user in guild's members
-      mutate(`/guild/${router.query.guild}`)
+      setTimeout(() => {
```

Comment:
Why do we need this setTimeout here? Do we need to wait for the 2 other mutations above (in that case we could maybe await those)?

Reply:
I think it was needed because of read replica lag. The mutations are needed because we might see some additional RRR-s after we joined a guild, but when I tried without timeout, I received the same older data